enwikuna/wordpress-playground-plugin-loader
# Add development environment

This PR adds a local development environment and testing instructions.

The goal is to make it simple for developers to get started with the project.

## Testing instructions

- checkout this branch
- follow [the Development instructions](https://github.com/enwikuna/wordpress-playground-plugin-loader/pull/3/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R20)
- ensure that you have a working dev environment
